tests/bluetooth/controller/ctrl_data_length_update/Kconfig

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2022, Nordic Semiconductor ASA
+
+config BT_CTLR_PHY
+	prompt "Bluetooth 5.0 phy"
+	help
+	  This is generally a hidden symbol but for unit testing a prompt is
+	  defined to allow controlling this setting in the build.
```

Comment:
@thoh-ot the fix here in this commit 4e90b63af06bc936b4640cc8a5628c2dab9a753b is not necessarily the cleanest approach.

I believe the proper approach would be to create a `prj.conf` with needed settings similar to this commit: https://github.com/zephyrproject-rtos/zephyr/pull/48732/commits/e58fab9d739611045d9b8b5d6ce9df62a15d0a73
but the `BT_CTLR_PHY` has indirect dependencies to many settings, including `BT_LLL_VENDOR_NORDIC` or `BT_LLL_VENDOR_OPENISA` so unit testing would have to specify one of those symbols.

So in the end, the cleanest solution for unit testing might actually be to bypass dependencies for settings as is done in this commit.

Reply:
@tejlmand Been working with the PR today, but havn't gotten a conclusion yet. There is some trickery in the `tests/bluetooth/controller/common/defaults_cmake.txt` that i'm working on removing and that gives me a tons of compile errors due to all this missing Kconfig symbols, so i'm trying to work around that.